app/models.py

```diff
@@ -1084,11 +1085,7 @@ def process_type(cls):
         )
 
     def queue_to_use(self):
-        return {
-            NORMAL: QueueNames.NORMAL,
-            PRIORITY: QueueNames.PRIORITY,
-            BULK: QueueNames.BULK,
-        }[self.process_type]
```

Comment:
Essentially, we move this logic to app.utils to avoid circular import errors 😞

Reply:
Makes sense!